r/R/dplyr-functions.R

```diff
@@ -398,6 +398,22 @@ nse_funcs$str_split <- function(string, pattern, n = Inf, simplify = FALSE) {
   )
 }
 
+nse_funcs$pmin <- function(..., na.rm = FALSE) {
+  Expression$create(
```

Comment:
This will allow users to pass literal scalar arguments to `pmin()` in dplyr:
```suggestion
  build_expr(
```